dist/server/index.js

```diff
@@ -2,15 +2,16 @@ const find = require('lodash/find');
 const xmlParser = require('fast-xml-parser');
 
 async function setLaunchUrl(element, { config, storage, storageProxy }) {
-  const { protocol, path } = config.storage;
+  const { protocol } = config.storage;
   const { root, assets } = element.data;
   if (!root) return element;
   const manifestKey = find(assets, it => it.endsWith('imsmanifest.xml'));
   const manifest = await storage.getFile(manifestKey.substr(protocol.length, manifestKey.length));
   const options = { ignoreAttributes: false, attributeNamePrefix: '$_' };
   const { manifest: parsedManifest } = xmlParser.parse(manifest.toString(), options);
   const { $_href: resourcePath } = parsedManifest.resources.resource;
-  const launchKey = `${path}/${root}/${resourcePath}`;
+  const prefix = storage.getAssetsPath(element.repositoryId);
```

Comment:
Can we find a more consistent name for this method (with the rest of storage methods/context)?
Maybe `getStoragePath` or `getPath`?